edx_sga/tests/test_sga.py

```diff
@@ -728,7 +728,7 @@ def test_prepare_download_submissions(
             {
                 'submission_id': uuid.uuid4().hex,
                 'filename': "test_{}.txt".format(uuid.uuid4().hex),
-                'timestamp': datetime.datetime.utcnow()
+                'timestamp': datetime.datetime.now(tz=pytz.utc)
```

Comment:
what is difference between utcnow() and now(tz=pytz.utc)?

Reply:
![ss 2017-12-18 at 10 31 04](https://user-images.githubusercontent.com/14932219/34113703-8fb1f40a-e3de-11e7-9049-16781005538c.png)